Containerfile.rhel

```diff
@@ -0,0 +1,36 @@
+FROM --platform=${BUILDPLATFORM:-linux/amd64} registry.access.redhat.com/ubi9 
```

Comment:
What does the `--platform` switch accomplish? (I did review the docs so I know what it does.)

Reply:
It's for eventual multi-arch support. I had looked into it and added this, we only have amd64 rpms right now though anyway.

Reply:
Considering that, I'll just remove it for now. I can add it back in if/when we need to.